src/content/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "NerdGraph tutorial: Browser agent monitoring examples"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+  - Browser
+  - Mobile
+metaDescription: Examples of using New Relic NerdGraph API to adjust a browser agent
+---
+
+You can create browser applications in the NerdGraph API instead of using the UI. The advantage to this is that you can programmatically create and retrieve the JavaScript snippet to copy and paste into your browser application.
```

Comment:
```suggestion
You can create browser applications in the NerdGraph API instead of using the UI. The advantage to this is that when it's time to instrument your browser app with New Relic, you can programmatically create and retrieve the JavaScript snippet to copy and paste into your browser app.
```